File: Python/3190_find_minimum_operations_to_make_all_elements_divisible_by_three.py
class Solution:
    def minimumOperations(self, nums: list[int]) -> int:
        operations = 0
        for num in nums:
            if num % 3 == 0:
                continue
            elif (num - 1) % 3 == 0:
                operations += 1
            elif (num + 1) % 3 == 0:
                operations += 1
        return operations
        

File: JavaScript/3190_find_minimum_operations_to_make_all_elements_divisible_by_three.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minimumOperations = function(nums) {
    let operations = 0;
    for (const num of nums) {
        if (num % 3 !== 0) {
            operations += Math.min(num % 3, 3 - (num % 3));
        }
    }
    return operations;
};

